cellmlmanip/mathml2sympy/transpiler.py

```diff
@@ -236,72 +217,32 @@ def root_handler():
     """
     https://www.w3.org/TR/MathML2/chapter4.html#contm.root
     operator taking qualifiers
-    TODO: implement <degree>
-    """
-    def __wrapped_root(radicand, degree=None):
-        if degree is None:
-            # by default, sqrt
-            return sympy.root(radicand, 2)
-        else:
-            raise NotImplementedError
-            # return sympy.root(b, a)
-    return __wrapped_root
-
-
-# RELATIONS ####################################################################################
-
-def eq_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.eq
-    n-ary operator
-    """
-    return sympy.Eq
-
-
-def leq_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.leq
-    """
-    return sympy.Le
-
-
-def lt_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.lt
-    """
-    return sympy.Lt
-
-
-def geq_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.geq
-    n-ary relation
-    """
-    return sympy.Ge
-
-
-def gt_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.gt
-    n-ary relation
-    """
-    return sympy.Gt
-
 
-def and_handler():
-    """
-    https://www.w3.org/TR/MathML2/chapter4.html#contm.and
-    n-ary operator
-    """
-    return sympy.And
+    Nasty:
+    The root element is used to construct roots. The kind of root to be taken is specified by a
+    degree element, which should be given as the second child of the apply element enclosing the
+    root element. Thus, square roots correspond to the case where degree contains the value 2, cube
+    roots correspond to 3, and so on. If no degree is present, a default value of 2 is used.
+    """
+    def __wrapped_root(first_argument, second_argument=None):
```

Comment:
I'm not sure there's any benefit to using `__` here - it's not a class method that needs to be private, just a local variable that gets returned, at which point the name doesn't matter.

Reply:
Two underscores is reserved for python internals anyway, so should be a single underscore if anything

Reply:
Cool - single underscore silences warnings about docstrings in my IDE!